lib/atomic-management.js

```diff
@@ -25,25 +25,28 @@ const mergeJSON = require("merge-json") ;
 const CONFIG_FILE_PREFIX = '.atom/config'
 import { CompositeDisposable, BufferedProcess } from 'atom';
 
+import RootController from './views/RootController';
```

Comment:
Weird error I got when trying to run but should it be "./views/rootController"? JS importing syntax seems to work that way (at least for me)